packages/browser/src/core/user/index.ts

```diff
@@ -398,3 +398,53 @@ export class Group extends User {
     return undefined
   }
 }
+
+export class UniversalStorage {
+  private cookies: Store
+  private localStorage: Store
```

Comment:
Looking at this file, I feel like it would make sense to create an actual interface and implement it where it makes sense, makes creating testing stubs so much easier and ensuring unity between all these similar storage classes.